.gitignore

```diff
@@ -8,4 +8,5 @@ eclipse
 .project
 .settings
 bin
-.metadata
\ No newline at end of file
+.metadata
+*.bak
```

Comment:
Might want to remove this, we don't want .bak files in the actual mod!

Reply:
Notepad++ made a .bak file when I was adding something to a lang file. This prevents all of my .bak files from getting in the repository because I don't want to manually delete them all on my end. bak files can be ignored using build.gradle as well.

Reply:
Gotcha, no problem here then!